app/templates/request-common-people-in-household.html

```diff
@@ -0,0 +1,76 @@
+{%- set field_messages_dict=dict(messages_dict['ERROR']|groupby('field')) -%}
+
+{%- from 'components/question/_macro.njk' import onsQuestion -%}
+{%- from 'components/button/_macro.njk' import onsButton -%}
+{%- from 'components/input/_macro.njk' import onsInput -%}
+
+{%- set paper_questionnaire_link = url('CommonEnterAddress:get', display_region=display_region, user_journey='requests', sub_user_journey='paper-questionnaire') -%}
+
+{%- set form =  {
+    'method': 'POST',
+    'attributes': {
+        'action': url('RequestCommonPeopleInHousehold:post', display_region=display_region, request_type=request_type)
+    }
+} -%}
+
+{%- if 'number_of_people_empty' in field_messages_dict -%}
+    {%- set error_number_of_people = {'id': 'number_of_people_empty', 'text': _('Enter the number of people in your household')} -%}
+{%- elif 'number_of_people_nan' in field_messages_dict -%}
+    {%- set error_number_of_people = {'id': 'number_of_people_nan', 'text': _('Enter a numeral')} -%}
+{%- elif 'number_of_people_continuation' in field_messages_dict -%}
+    {%- if display_region == 'ni' -%}
+        {%- set error_number_of_people = {'id': 'number_of_people_continuation', 'text': _('Enter a number greater than 6')} -%}
+    {%- else -%}
+        {%- set error_number_of_people = {'id': 'number_of_people_continuation', 'text': _('Enter a number greater than 5')} -%}
+    {%- endif -%}
+{%- endif -%}
+
+{%- if request_type == 'continuation-questionnaire' -%}
+    {%- set question_description = _('<p>This is to make sure we send you the right number of continuation questionnaires if there are more than 5 people in your household</p>') -%}
+{%- else -%}
+    {%- set question_description = _('<p>This is to make sure we send you the right number of paper questionnaires</p>') -%}
+{%- endif -%}
+
+{%- block main -%}
+
+    {%- if messages_dict -%}
+        {%- include 'partials/messages.html' with context -%}
+    {%- endif -%}
+
+    {%- call onsQuestion({
+        'title': _('How many people are in your household?'),
+        'description': question_description
+    }) -%}
+
+        {{
+            onsInput({
+                'id': 'number_of_people',
+                'label': {
+                    'text': _('Number of people')
+                },
+                'value': form_value_number_of_people,
+                'name': 'number_of_people',
+                'error': error_number_of_people,
+                'classes': 'input--w-2'
+            })
+        }}
+
+    {%- endcall -%}
+
+    {{
+        onsButton({
+            'text': _('Continue'),
+            'classes': 'u-mt-l'
+        })
+    }}
+
+    {%- if request_type == 'continuation-questionnaire' -%}
+        {%- autoescape false -%}
+            <p class="u-mt-xl">{{ _('If you don’t already have a paper census for your household, you will need to %(open)srequest a household paper questionnaire%(close)s.', open='<a href="%s">' % paper_questionnaire_link, close='</a>')|safe }}</p>
```

Comment:
The requirement is a little vague here (the design doesn't specify where the link is going) - I'll leave as is, but may revisit (there are a couple of issues to resolve around session killing or continuing, so it will drop out of that I expect)